apiserver/debuglog.go

```diff
@@ -200,6 +202,17 @@ func readDebugLogParams(queryMap url.Values) (*debugLogParams, error) {
 		params.filterLevel = level
 	}
 
+	if value := queryMap.Get("startTime"); value != "" {
+		startTime, err := time.Parse(time.RFC3339Nano, value)
+		if err != nil {
+			return nil, errors.Errorf("start time %q is not a valid time in RFC3339 format", value)
+		}
+		if startTime.After(time.Now()) {
```

Comment:
I'm a bit sceptical about this one. What if the controller's clock currently wrong and it has logs from the "future"?